apps/dash-multipage-report/app.py

```diff
@@ -0,0 +1,2738 @@
+                                    ],
+                                    style={
+                                        "border-left": "5px",
+                                        "border-left-style": "solid",
+                                        "border-left-color": color_2,
+                                        "border-left-width": "7px",
+                                        "padding-left": "20px",
+                                        "margin-top": "20px",
+                                    },
+                                ),
+                            ],
+                            style={
+                                "display": "block",
+                                "float": "right",
+                                "width": "400px",
+                                "margin-left": "10px",
+                                "margin-right": "15px",
+                            },
+                        ),
+                    ],
+                    className="subpage",
+                )
+            ],
+            className="page",
+        ),
+        ## Page 2
+        html.Div(
+            [
+                html.Div(
+                    [
+                        html.Div(
+                            [html.H1("LOREM IPSUM")],
+                            style={
+                                "background-color": color_1,
+                                "color": "white",
+                                "width": "200px",
+                                "height": "160px",
+                                "padding": "20px",
+                                "margin-left": "50px",
+                                "margin-top": "-38px",
+                            },
+                        ),
+                        html.Div(
+                            [
+                                html.P(
+                                    lorem.paragraph() * 3, style={"margin-top": "40px"}
+                                ),
+                                html.P(
+                                    lorem.paragraph() * 2, style={"margin-top": "20px"}
+                                ),
+                                html.P(
+                                    lorem.paragraph() * 2, style={"margin-top": "20px"}
+                                ),
+                            ],
+                            style={"margin-left": "80px", "margin-right": "40px"},
+                        ),
+                        html.Div(
+                            [
+                                html.P(
+                                    lorem.paragraph() * 2, style={"margin-top": "40px"}
+                                ),
+                                html.P(
+                                    lorem.paragraph() * 3, style={"margin-top": "20px"}
+                                ),
+                            ],
+                            style={"margin-left": "80px", "margin-right": "40px"},
```

Comment:
This is used twice in line 295. You can save this in a css `class` to get rid of some inline styling.